README.md

```diff
@@ -1,57 +1,253 @@
 # Rails form_tag
 
-## Objectives
 
-1. Build a functional Rails form using just HTML
-2. Describe the purpose of the CSRF and how to embed it manually
-3. Correctly assign a form action and form method.
-4. Use a route helper as the value of a form tag's action
-5. Map a form submission to a POST request to a controller/action
-6. Use HTML <input> in a form
-7. Explain how the input's name attribute corresponds to params
-8. Raise params.inspect to test a form submission
-9. Build a functional Rails form using form_tag
-10. Pass a route helper as the argument to form_tag
-11. Pass an options hash with method to form_tag
-12. Use text_field_tag and other form controls to create inputs
+## Rails Forms
 
+When it comes to forms in Rails you will discover that you will have the flexibility to utilize:
 
-## Notes
+* Ruby form gems
 
-building forms in rails, first lets get the posts#new action setup
+* Built in form helper methods
 
-new will render form.html.erb
+* Plain HTML form elements
 
-lets build the form with vanilla html
+This lesson is going to begin with integrating HTML form elements and then slowly start to refactor the form using Rails methods. It would be very easy to integrate form helpers and we could have our form working in a few minutes, however to fully understand what Rails is doing behind the scenes is more important than getting this form working, so we're going to build the system from the ground up so when we're finished you should be able to understand all of the processes that are necessary in order to process forms in an application properly and securely.
 
-most important part of a form is the action, where does this form submit to?
 
-we want the form to submit to POST /posts, let's draw that route, it's going to map to posts#create
+## Rendering the form view
 
-lets use the posts_path helper to population the form tag action.
+Let's first create a Capybara spec to ensure that going to ```posts/new``` takes us to our form:
```

Comment:
Last, I'd remind them what the `new_post_path` method will return. All these route helpers will be new